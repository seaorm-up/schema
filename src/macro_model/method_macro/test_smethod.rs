#![cfg(test)]

use crate::*;
smethod!(
    format("create app CONTENT {}", instance, instance),
    App,
    create_app // [get_app][get_app]
);
// delete would return [] all ways
// create as test would return value, so it is no need aa get_app
smethod!(
    format("delete app where name='{}'", instance, instance.name),
    App,
    delete,
    [create_app][get_app]
);

smethod!(
    format(
        "select * from app where name = '{}'",
        instance,
        instance.name
    ),
    App,
    get_app
);
