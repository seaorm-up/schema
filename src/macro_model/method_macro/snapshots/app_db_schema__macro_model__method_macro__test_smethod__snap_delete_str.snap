---
source: src/macro_model/method_macro/test_smethod.rs
expression: "format!(\"delete app where name='{}'\", instance.name)"
---
"delete app where name='adf'"
